test/stateTransitions.js

```diff
@@ -56,7 +53,7 @@ const checkState = async (auctionIndex, auctionStart, sellVolumesCurrent, sellVo
 
 const getState = async (contracts, ST, BT) => {
```

Comment:
@josojo Why are you passing in `contracts` here? It is in the upper scope but most importantly you are not using it in the actual fn block. Go ahead and remove this unless there's something I'm not seeing ;)